cdc/api/v2/changefeed.go

```diff
@@ -133,6 +135,28 @@ func (h *OpenAPIV2) createChangefeed(c *gin.Context) {
 		return
 	}
 
+	// cannot create changefeed if there are running lightning/restore tasks
+	tlsCfg, err := toolutils.ToTLSConfig(
```

Comment:
Why not use `pkg/security` to generate tls configuration here？

Reply:
Didn't notice that we have existing method in `pkg/security`. Have swtiched to `ToTLSConfig` defined in `pkg/security`